ci-operator/step-registry/servicemesh/istio/int-tests/servicemesh-istio-int-tests-commands.sh

```
@@ -22,23 +22,64 @@ run_tests() {
+    istio)
+      PARSE_TEST_CONFIG_MIDSTREAM_VARIANT="midstream_helm"
+      ;;
+    *)
+      echo "Unsupported CONTROL_PLANE_SOURCE: ${CONTROL_PLANE_SOURCE} (expected istio or sail)" >&2
+      exit 1
+      ;;
+  esac
+
+  ./parse-test-config.sh ${CONFIG_FILE} "${TEST_SUITE}" "${PARSE_TEST_CONFIG_MIDSTREAM_VARIANT}"
+  # it will eval SKIP_PARSER_SKIP_TESTS and SKIP_PARSER_SKIP_SUBSUITES ENV variables which will be used in the prow/integ-suite-ocp.sh
+  eval $(./parse-test-config.sh "${CONFIG_FILE}" "${TEST_SUITE}" "${PARSE_TEST_CONFIG_MIDSTREAM_VARIANT}")
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)